janusgraph-core/src/main/java/org/janusgraph/graphdb/database/serialize/StandardSerializer.java

```diff
@@ -138,6 +138,14 @@ public StandardSerializer() {
     public synchronized <V> void registerClass(int registrationNo, Class<V> datatype, AttributeSerializer<V> serializer) {
         Preconditions.checkArgument(registrationNo >= 0 && registrationNo < MAX_REGISTRATION_NO, "Registration number" +
                 " out of range [0,%s]: %s", MAX_REGISTRATION_NO, registrationNo);
+
+        if (datatype == HashMap.class)
+        {
```

Comment:
Write on one line (e.g. `if (...) {`) for consistency with style used elsewhere in class.